src/cicd/utils.ts

```diff
@@ -0,0 +1,94 @@
+import { Effect, PolicyStatement } from 'aws-cdk-lib/aws-iam';
+
+export interface getCodeartifactProps {
+  partition: string;
+  region: string;
+  account: string;
+  domain: string;
+  repository: string;
+}
+
+export function getCodeartifactReadPolicyStatements(
```

Comment:
Minor nitpick: Shouldn't `artifact` be capitalized? Same for the props interface.